### PR TITLE
Fail with HTTP 409 with empty response when creating a second UK

### PR DESCRIPTION
**Pull request by dabd**
property for a nino

### CODE REVIEW

File: app/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResource.scala [hunk 1]
```diff
@@ -50,8 +48,7 @@ object PropertiesResource extends BaseController {
             case false => InternalServerError
           }
 
-        case Left(err) =>
-          Conflict(Json.toJson(Errors.businessError(err))).withHeaders(LOCATION ->  s"/self-assessment/ni/$nino/uk-properties")
+        case Left(_) => Conflict
```
Comment (GlassAndOneHalf) [inline app/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResource.scala]: I think we still require the LOCATION header to be there.

File: func/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResourceSpec.scala [hunk 2]
```diff
@@ -31,8 +31,6 @@ class PropertiesResourceSpec extends BaseFunctionalSpec {
         .post(Jsons.Properties()).to(s"/ni/$nino/uk-properties")
         .thenAssertThat()
         .statusIs(409)
-        .bodyIsLike(Jsons.Errors.businessError("ALREADY_EXISTS" -> ""))
-        .responseContainsHeader("Location", s"/self-assessment/ni/$nino/uk-properties".r)
```
Comment (GlassAndOneHalf) [inline func/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResourceSpec.scala]: Restore this check.

Comment (GlassAndOneHalf) [inline func/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResourceSpec.scala]: Without the `bodyIsLike` part.

File: app/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResource.scala (hunk 1 above)
Comment (dabd) [inline app/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResource.scala]: fixed in https://github.com/hmrc/self-assessment-api/pull/280/commits/f04f922ed753e574ef42d430662abf1bcb48bc34

File: func/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResourceSpec.scala (hunk 2 above)
Comment (dabd) [inline func/uk/gov/hmrc/selfassessmentapi/resources/PropertiesResourceSpec.scala]: fixed in https://github.com/hmrc/self-assessment-api/pull/280/commits/f04f922ed753e574ef42d430662abf1bcb48bc34

**Pull request merged**